frontend/src/components/TopBar/TopBar.jsx

```diff
@@ -23,8 +23,8 @@ export default function TopBar() {
         Roadie
       </Typography>
       <div className={styles.navItems}>
-        <Button>Join Existing Group</Button>
-        <Button>Create New Group</Button>
+        <Button>Join Existing Trip</Button>
```

Comment:
With the new change of hosts or organisers adding people to trips, we won't need this button. But this can be easily fixed later.